script.js

```diff
@@ -1 +1,27 @@
-// your code
+let center=document.querySelector(".section-center")
```

Comment:
1. const would be better.
2. Please use space before and after any math operator.
3. Use better name

const  center = document.querySelector(".section-center")

Reply:
1. updated let to const and gave more meaning full name
2. added spaces per recommendation